src/libserver/redis_pool.cxx

```diff
@@ -442,6 +498,21 @@ redis_pool_connection::redis_pool_connection(redis_pool *_pool,
 	}
 }
 
+static bool ensure_ssl_inited()
+{
+    static bool inited = false;
+    if (!inited) {
+        if (redisInitOpenSSL() == REDIS_OK) {
```

Comment:
We actually don't need it - SSL is guaranteed to be initialised before that, so the whole function can be safely removed.